attributes/README.md

```diff
@@ -0,0 +1,37 @@
+# Zarr Attributes Specifications
+
+This directory contains detailed specifications for Zarr attribute extensions that are registered in the [zarr-extensions repository](https://github.com/zarr-developers/zarr-extensions).
+
+## Overview
+
+Zarr attributes are metadata stored in the `attributes` dictionary of Zarr arrays and groups. While the `attributes` dictionary can contain arbitrary metadata, this directory provides specifications for commonly used attribute conventions within the Zarr community.
+
+## Registered Attributes
+
+### [`geo/proj`](./geo/proj/README.md) - Geo Projection Attribute Extension
+
+**Status**: Proposed  
+**Registry Entry**: [zarr-extensions/attributes/geo/proj](https://github.com/zarr-developers/zarr-extensions/tree/main/attributes/geo/proj)
+
+Defines a JSON object that encodes datum and coordinate reference system (CRS) information for geospatial data stored under the `proj` key within the `geo` dictionary.
+
+**Key Features**:
+- Simple, standardized CRS encoding without complex nested structures
+- Compatible with existing geospatial tools (GDAL, rasterio, pyproj)
+- Based on the proven STAC Projection Extension model
+- Group-to-array inheritance model for CRS metadata
+- Support for multiple CRS representations (EPSG codes, WKT2, PROJJSON)
+
+**Specification**: [./geo/proj/README.md](./geo/proj/README.md)
+
+## Contributing
+
+For changes to attribute specifications:
+
+1. **Content Changes**: Submit PRs to this repository for specification content
+2. **Registration Changes**: Submit PRs to [zarr-extensions](https://github.com/zarr-developers/zarr-extensions) for registry updates
+3. **New Attributes**: Follow the [registration process](https://github.com/zarr-developers/zarr-extensions#registering-an-attribute) in zarr-extensions
```

Comment:
_This_ seems like the right place to mention `zarr-extensions`, because people contributing new specs to EOPF-Explorer/data-model should also consider registering those specs at `zarr-extensions`

Reply:
we might also indicate that every specification should have an _implementation_ defined in this repo. This ensures that the spec can actually be implemented.